src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -303,6 +303,8 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 	<_InstantRunEnabled Condition=" '$(_InstantRunEnabled)' == '' ">False</_InstantRunEnabled>
 	<_AndroidBuildPropertiesCache>$(IntermediateOutputPath)build.props</_AndroidBuildPropertiesCache>
 
+	<JniMarshalMethods Condition=" '$(JniMarshalMethods)' == '' ">False</JniMarshalMethods>
```

Comment:
By "convention" (*mostly* followed, but not everywhere), this property should begin with `Android`.

Additionally, this name doesn't convey *intent*. *What about* JNI Marshal Methods? Will this property eventually control *generation* as well as preservation, or just linker preservation?

Assuming we want it to control the *entire* `jnimarshalmethod-gen.exe` stack, I think we should call this `$(AndroidGenerateJniMarshalMethods)`.

Whatever it's called, this property should also be documented in https://github.com/xamarin/xamarin-android/blob/master/Documentation/guides/BuildProcess.md.